packages/datastore/package.json

```diff
@@ -18,21 +18,30 @@
   "files": [
     "dist/*",
     "src/*",
-    "types/*"
+    "types/*",
+    "types-3.4/*"
   ],
   "main": "dist/index.js",
   "module": "dist/index.es6",
   "unpkg": "dist/index.min.js",
   "jsdelivr": "dist/index.min.js",
   "types": "types/index.d.ts",
+  "typesVersions": {
```

Comment:
Wow, I didn't know this was a thing! https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html#version-selection-with-typesversions

Reply:
Been there a long time (which is good as its the type of thing you only need way after its been released!)